cpu/stm32/include/candev_stm32.h

```diff
@@ -42,7 +42,7 @@ extern "C" {
 #elif defined(CPU_FAM_STM32F1) || defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4)
 #define CANDEV_STM32_CHAN_NUMOF 2
 #elif defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F3) || \
-      defined(CPU_FAM_STM32L4) || DOXYGEN
+      defined(CPU_FAM_STM32L4) || defined(CPU_LINE_STM32F722xx) || DOXYGEN
```

Comment:
Why not
```suggestion
      defined(CPU_FAM_STM32L4) || defined(CPU_FAM_STM32F7) || DOXYGEN
```

Reply:
See https://github.com/RIOT-OS/RIOT/pull/16150#discussion_r588613064

Reply:
Ah I see - how about we do

```C
#if defined(CAN3)
#define CANDEV_STM32_CHAN_NUMOF 4
#elif defined(CAN2)
#define CANDEV_STM32_CHAN_NUMOF 3
#elif defined(CAN1)
#define CANDEV_STM32_CHAN_NUMOF 2
#elif defined(CAN0)
#define CANDEV_STM32_CHAN_NUMOF 1
#else
#error "CAN STM32: CPU not supported"
#endif
```

as a follow-up?